src/solidity-frontend/solidity_convert.cpp

```diff
@@ -3299,7 +3299,10 @@ bool solidity_convertert::get_function_definition(
 
   // 10. Add symbol into the context
   symbolt &added_symbol = *move_symbol_to_context(symbol);
-
+  // 10.1 set a code_blockt as the placeholder in advane
+  // to support the recursive function call
+  added_symbol.type = type;
+  added_symbol.value = code_blockt();
```

Comment:
I don't think you need to do this. We will set it in https://github.com/esbmc/esbmc/blob/a50d7155ce60ed752f0f6508ca6cf8b044661584/src/solidity-frontend/solidity_convert.cpp#L3348 and
https://github.com/esbmc/esbmc/blob/a50d7155ce60ed752f0f6508ca6cf8b044661584/src/solidity-frontend/solidity_convert.cpp#L3377

Reply:
@prototypeC14